src/components/carousel/carousel.component.js

```diff
@@ -149,10 +148,17 @@ class BaseCarousel extends React.Component {
   }
 
   /** Gets the currently visible slide */
-  visibleSlide() {
+  visibleSlide = () => {
     let index = this.state.selectedSlideIndex;
 
-    const visibleSlide = compact(React.Children.toArray(this.props.children))[index];
+    const visibleSlide = (
+      <CSSTransition
+        className='carbon-carousel__transition'
+        classNames={ this.transitionName() }
+        timeout={ { enter: TRANSITION_TIME, exit: TRANSITION_TIME } }
+      >{compact(React.Children.toArray(this.props.children))[index]}
```

Comment:
Formatting - put `compact...` in new row

Reply:
Updated in https://github.com/Sage/carbon/pull/2496/commits/ac367dc8ea8d2c0b8524b08a78f38b0d13800f4e